packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -249,6 +274,10 @@ export const getTxExplorer = (chainId?: ChainId | number, txHash?: string) => {
     case 1:
       return `https://etherscan.io/tx/${txHash}`;
 
+    case ChainId.PGN_TESTNET_CHAIN_ID:
+    case 58008:
```

Comment:
Not for this PR but these strange looking `case` statements could be simplified by having `switch (String(chainId)) {` at the top, I suppose.

Reply:
Even better - no need for a switch statement. We are not doing any logic, just getting the right txExplorer url. Two things can be improved here:
- type safety: same method as https://github.com/gitcoinco/grants-stack/blob/builder-typesafe-chainid/packages/builder/src/contracts/deployments.ts
- get explorer urls from wagmi/viem - just for simplicity.